src/sentry/integrations/vsts/issues.py

```diff
@@ -79,27 +84,39 @@ def get_work_item_choices(self, project, group):
         item_tuples = list(item_type_map.items())
 
         # try to get the default from either the last value used or from the first item on the list
-        defaults = self.get_project_defaults(group.project_id)
+        defaults = {}
+        if group:
+            defaults = self.get_project_defaults(group.project_id)
         try:
             default_item_type = defaults.get("work_item_type") or item_tuples[0][0]
         except IndexError:
             return None, item_tuples
 
         return default_item_type, item_tuples
 
+    def get_create_issue_config_no_group(self, project):
+        return self.get_create_issue_config(None, None, project=project)
+
     def get_create_issue_config(self, group, user, **kwargs):
         kwargs["link_referrer"] = "vsts_integration"
-        fields = super(VstsIssueSync, self).get_create_issue_config(group, user, **kwargs)
-        # Azure/VSTS has BOTH projects and repositories. A project can have many repositories.
-        # Workitems (issues) are associated with the project not the repository.
-        default_project, project_choices = self.get_project_choices(group, **kwargs)
+        fields = []
+        if group:
+            fields = super(VstsIssueSync, self).get_create_issue_config(group, user, **kwargs)
+            # Azure/VSTS has BOTH projects and repositories. A project can have many repositories.
+            # Workitems (issues) are associated with the project not the repository.
+            default_project, project_choices = self.get_project_choices(group, **kwargs)
+        else:
+            default_project, project_choices = self.get_project_choices(**kwargs)
```

Comment:
@ceorourke If group is `None`, you can still call this with `default_project, project_choices = self.get_project_choices(group, **kwargs)`